validator/lib/validateTileset.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+var Promise = require('bluebird');
+
+module.exports = validateTileset;
+
+/**
+ * Checks whether the geometricError of each tile is <= the geometric error of its parent tile.
+ *
+ * @param {JSON} The JSON object representing the tileset.
+ * @return {Promise} A promise that resolves with two parameters - (1) a boolean for whether the tileset is valid
+ *                                                                 (2) the error message if the tileset is not valid.
+ *
+ */
+function validateTileset(tileset) {
+    if (tileset.root.geometricError > tileset.geometricError) {
+        return Promise.resolve(false, "Root has geometricError greater than tileset");
+    }
+
+    return new Promise(function(resolve, reject) {
```

Comment:
As we talked about offline, this approach is ok for now but may need some adjustments once we actually start to validate individual tiles. You can remove the `reject` parameter for now since it isn't used.